src/main/java/org/teleight/teleightbots/bot/manager/BotManagerImpl.java

```diff
@@ -71,7 +73,7 @@ private <T extends TelegramBot> void startProcessor(@NotNull T telegramBot, @Not
                     // Happens when the bot token or the username is invalid.
                     // It can also happen if the webhook was considered invalid by the bot API.
                     if (throwable != null) {
-                        System.out.println("An error occurred while authenticating the bot " + telegramBot.getBotUsername() + ": " + throwable.getMessage());
+                        log.error("An error occurred while authenticating the bot {}: {}", telegramBot.getBotUsername(), throwable.getMessage());
```

Comment:
I feel like there should be some kind of fallback logging when it comes to error handling.

If you try running a bot with a misconfigured token or username, without a valid slf4j implementation, the program will just silently exit without telling you what went wrong.

Or we could make the implementation required in order to use the library, even though I do not quite like this idea.